README.md

```diff
@@ -257,6 +257,111 @@ window.navigator.getUserMedia({
 });
 ```
 
+**List of available sources:**
```

Comment:
I think this is in the screensharing section, but could you change that to "List of available screensharing sources" or "List of valid screensharing sources"